package.json

```diff
@@ -206,6 +206,11 @@
 				"type": "boolean",
 				"default": false,
 				"description": "Turn on this setting to stop this extension from modifying the VS Code configuration of the workspace. Note that this will also disable some features related to embedded Python and Shell code."
+			},
+			"bitbake.enableCodeLensReferencesOnFunctions": {
+				"type": "boolean",
+				"default": false,
```

Comment:
this surprised me also. However since currently we only have code lens for references, which have their dedicated button, I'm not sure the feature adds much value right now. Meanwhile it would bloat the interface.

Reply:
This demo is cool however I think codelenses are justified when we have specific commands to run. There are few extensions doing so and the additional buttons could annoy users. A useful example for instance is the git extension which adds additional commands for comparing with other branches.

Reply:
I think we would need different commands in order for this to be useful. I cannot think of one that isn't already seemlessly integrated into the interface yet (value hovers, rename, definitions, references...).